Adding template information to create a repository section (#37)
empa-scientific-it/how-to-git: src/content/docs/01-repository.mdx

3. Make sure the **visibility** is set to _Public_ (default)

> 4. Start with a template: Leave it set to _No template_ (default)

Reviewer: ```suggestion
4. Leave the rest of the options set to default
```

5. **Uncheck** the option _Initialize this repository with a README_

6. Add `.gitignore`: Leave it set to _No .gitignore_ (default)

> 7. Choose a license: Leave it set to _No license_ (default)

Reviewer: Please delete these lines

> 8. Click on the green button "Create repository"

Reviewer: ```suggestion
5. Click on the green button "Create repository"
```
